docs/pages/includes/install-linux.mdx

```diff
@@ -1,220 +1,186 @@
+  $ cd teleport
+  $ sudo ./install
+  ```
 
-  <TabItem label="Amazon Linux 2/RHEL Legacy (RPM)">
-        ```code
-        $ sudo yum-config-manager --add-repo https://rpm.releases.teleport.dev/teleport.repo
-        $ sudo yum install teleport
-
-        # Optional:  Using DNF on newer distributions
-        # $ sudo dnf config-manager --add-repo https://rpm.releases.teleport.dev/teleport.repo
-        # $ sudo dnf install teleport
-        ```
   </TabItem>
-
 </Tabs>
 </ScopedBlock>
 <ScopedBlock scope={["enterprise"]}>
+<Tabs>
+  <TabItem label="Debian/Ubuntu (DEB)">
 
-Visit the [Downloads Page](https://dashboard.gravitational.com/web/downloads) in
-the customer portal and select the URL for your package of choice.
+  Aftter Downloading the `.deb` file for your system architecture, install it with
+  `dpkg`. The example below assumes the `root` user:
 
-Next, use the appropriate commands for your environment to install your package.
+  ```code
+  $ dpkg -i ~/Downloads/teleport-ent_(=teleport.version=)_<Var name="$SYSTEM-ARCH" description="The CPU architecture of the system Teleport will be installed on"/>.deb
+  Selecting previously unselected package teleport-ent.
+  (Reading database ... 30810 files and directories currently installed.)
+  Preparing to unpack teleport-ent_(=teleport.version=)_$SYSTEM_ARCH.deb ...
+  Unpacking teleport-ent (=teleport.version=) ...
+  Setting up teleport-ent (=teleport.version=) ...
+  ```
+
+  </TabItem>
+  <TabItem label="Amazon Linux 2/RHEL (RPM)">
+
+  After Downloading the `.rpm` file for your system architecture, install it with `rpm`:
 
-For example, use the following commands to install Teleport on a machine with an
-x86-64 chip via tarball:
-
-```code
-$ curl https://get.gravitational.com/teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz.sha256
-# <checksum> <filename>
-$ curl -O https://get.gravitational.com/teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz
-$ shasum -a 256 teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz
-# Verify that the checksums match
-$ tar -xvf teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz
-$ cd teleport-ent
-$ sudo ./install
-```
-
-For FedRAMP/FIPS-compliant installations of Teleport Enterprise, package URLs
-will be slightly different:
-
-```code
-$ curl https://get.gravitational.com/teleport-ent-v(=teleport.version=)-linux-amd64-fips-bin.tar.gz.sha256
-# <checksum> <filename>
-$ curl -O https://get.gravitational.com/teleport-ent-v(=teleport.version=)-linux-amd64-fips-bin.tar.gz
-$ shasum -a 256 teleport-ent-v(=teleport.version=)-linux-amd64-fips-bin.tar.gz
-# Verify that the checksums match
-$ tar -xvf teleport-ent-v(=teleport.version=)-linux-amd64-fips-bin.tar.gz
-$ cd teleport-ent
-$ sudo ./install
-```
+  ```code
+  $ rpm -i ~/Downloads/teleport-ent-(=teleport.version=).<Var name="$SYSTEM-ARCH"/>.rpm
+  warning: teleport-ent-(=teleport.version=).$SYSTEM-ARCH.rpm: Header V4 RSA/SHA512 Signature, key ID 6282c411: NOKEY
```

Comment:
I'm not familiar with RPM, but if there's away to do this that doesn't create a warning that would be preferable. Do we just need to verify it against Teleport's public key?

Reply:
I'm not familiar with RPM either, so how about I spin this into a separate issue to research?